### ISSUE TITLE
When will you upgrade the plugin to ElasticSearch v0.90+?

### ISSUE THREAD

**Issue by confile** [mstein/elasticsearch-grails-plugin#64]
When will you upgrade the plugin to ElasticSearch v0.90+?

**Comment by drmichaelgorski** [comment]
@mstein are you still working on this plugin?

**Comment by rgordeev** [comment]
Hello! The same question, are you going to develop this plugin further?

**Comment by mstein** [comment]
Hi, I'm only doing upgrade (of the ES lib) & a bit of maintenance at the moment, though not at a regular pace... 
Oh and there is a snapshot of the ES plugin v0.90.3 available on the grails plugin repository.
":elasticsearch:0.90.3.0-SNAPSHOT"

**Comment by drmichaelgorski** [comment]
It would be great if you would still work on the plugin, because elastic search is great.

**Comment by rgordeev** [comment]
Yes, it would be great. I'm new in Grails and it seems to me it's a popular framework enough with a large community, but there are only a few plugins for integration with search engines and servers such as Solr, ElasticSearch, SphinxSearch and so on but they are old and with no any support. As for "standart" grails search plugin it works with old Compass library only and that's not good. In Rails ecosystem the situation is much better :(

**Comment by rgordeev** [comment]
And thanks for your answer by the way ;) 
It gives me a hope the situations with search plugins will change :)
